src/app/cup/cup-rating-group/cup-rating-group.component.html

```diff
@@ -39,6 +39,13 @@ <h4>Рейтинг групи</h4>
 </div>
 <app-cup-rating-group-table *ngIf="cupRatingGroup" [cupRatingGroup]="cupRatingGroup" [highlightConfig]="highlightConfig">
 </app-cup-rating-group-table>
+<a
+   *ngIf="possiblePromotion"
```

Comment:
if `possiblePromotion` array is empty this condition is `true`. So we need to check if array contains at least one item:
```suggestion
   *ngIf="possiblePromotion?.length"
```